its/src/test/java/com/sonar/it/csharp/MultipleProjectsTest.java

```diff
@@ -68,13 +68,13 @@ public static void beforeAll() throws Exception {
 
   @Test
   void projectTypesInfoIsLogged() {
-    assertThat(buildResult.getLogs()).contains("Found 4 MSBuild C# projects: 2 MAIN projects. 2 TEST projects.");
+    assertThat(buildResult.getLogs()).contains("Found 2 MSBuild C# projects: 2 MAIN projects.");
```

Comment:
I've removed the test part for simplicity. Project type detection is asserted in `UnitTestProjectTypeProbingTest`